src/Protocol/Protocol18x.cpp

```diff
@@ -247,6 +257,16 @@ void cProtocol180::SendBlockChanges(int a_ChunkX, int a_ChunkZ, const sSetBlockV
 
 
 
+void cProtocol180::SendCameraSetTo(const cEntity & a_Entity)
```

Comment:
This packet allows the spectator to look through the eyes of a mob right? I'm not sure if this is the right name. Perhaps something like `SendSetMobView`